src/main/java/org/fcrepo/spec/testsuite/crud/HttpPut.java

```diff
@@ -157,13 +157,14 @@ public void httpPutUpdateDisallowedTriplesResponse(final String uri) {
 
         final String body2 = doGet(locationHeader).asString();
 
-        final String newBody = body2.replace("containedFolderSlug", "some-name");
+        final String newBody = body2.replace("containedFolderSlug", "conflicting-contained-resource");
```

Comment:
I am inclined to replace many of these `slug`s or hardcoded resource names with UUIDs, but this is a constructive step forward.